src/components/ToolMenu/Draw/index.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+
+import {
+  faDrawPolygon,
+  faGripLines,
+  faCircle,
+  faFont,
+  faSquare,
+  faPenToSquare,
+  faUpload,
+  faTrash
+} from '@fortawesome/free-solid-svg-icons';
+
+import {
+  FontAwesomeIcon
+} from '@fortawesome/react-fontawesome';
+
+import GeoJSON from 'ol/format/GeoJSON';
+
+import {
+  useTranslation
+} from 'react-i18next';
+
+import DeleteButton from '@terrestris/react-geo/dist/Button/DeleteButton/DeleteButton';
+import DrawButton from '@terrestris/react-geo/dist/Button/DrawButton/DrawButton';
+import ModifyButton from '@terrestris/react-geo/dist/Button/ModifyButton/ModifyButton';
+import SimpleButton from '@terrestris/react-geo/dist/Button/SimpleButton/SimpleButton';
+import ToggleGroup from '@terrestris/react-geo/dist/Button/ToggleGroup/ToggleGroup';
+import UploadButton from '@terrestris/react-geo/dist/Button/UploadButton/UploadButton';
+import {
+  useMap
+} from '@terrestris/react-geo/dist/Hook/useMap';
+import {
+  DigitizeUtil
+} from '@terrestris/react-geo/dist/Util/DigitizeUtil';
+
+import './index.less';
+
+interface DefaultDrawProps { }
+
+export interface DrawProps extends Partial<DefaultDrawProps> { }
+
+export const Draw: React.FC<DrawProps> = (): JSX.Element => {
+  const {
+    t
+  } = useTranslation();
+
+  const map = useMap();
+
+  const onGeoJSONUpload = (geoJSONFile: any ) => {
+    // console.log(geoJSONFile);
+    const type = geoJSONFile.type !== 'application/geojson' ? geoJSONFile.type : 'NOT SUPPORTED';
```

Comment:
Looks quite complicated to me, we might want to check for the supported type only an return if an unsupported one has been handed over.